echopype/preprocess/api.py

```diff
@@ -8,26 +8,26 @@
 
 
 def _check_range_uniqueness(ds):
-    """Check if range changes across ping in a given frequency channel."""
+    """Check if `echo_range` changes across ping in a given frequency channel."""
     return (
-        ds["range"].isel(ping_time=0).dropna(dim="range_bin")
-        == ds["range"].dropna(dim="range_bin")
+        ds["echo_range"].isel(ping_time=0).dropna(dim="range_bin")
+        == ds["echo_range"].dropna(dim="range_bin")
     ).all()
 
 
 def compute_MVBS(ds_Sv, range_meter_bin=20, ping_time_bin="20S"):
     """Compute Mean Volume Backscattering Strength (MVBS)
-    based on intervals of range and ping_time specified in physical units.
+    based on intervals of `echo_range` and ping_time specified in physical units.
```

Comment:
Change to double back quotes, ``echo_range``. Same thing in a couple of cases a few lines below.